src/styles/index.scss

```diff
@@ -1,3 +1,64 @@
+@import './variables';
+
+// Загальні стилі для сторінки
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  width: calc(7 * $day-size + 6 * $gap + 2 * $padding);
+  margin: 0 auto;
+  gap: $gap;
+  padding: $padding;
+}
+
+.calendar__day {
+  display: flex; /* Вирівнювання по центру */
+  justify-content: center; /* Вирівнювання по горизонталі */
+  align-items: center; /* Вирівнювання по вертикалі */
+  position: relative;
+  height: $day-size;
+  width: $day-size;
+  box-sizing: border-box;
+  background-color: $background-color;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: $font-size;
+  border: solid $border-size black;
+  transition:
+    background-color $animation-duration,
+    transform $animation-duration;
+
+  &:hover {
+    cursor: pointer;
+    transform: translateY(-20px);
```

Comment:
The translateY value is hardcoded. It's better to move it to the variables file and use the variable here. This will make your code more maintainable.